1-exercises/B-setting-values/exercise1.js

```diff
@@ -24,8 +32,24 @@ let capitalCities = {
 
 // write code here
 
+let capitalCities = {
+  UnitedKingdom: {
+    name: "London",
+    population: 20,
+  },
+  China: {
+    name: "Beijing",
+    population: 21500000,
+  },
+Peru: {
+ name: "Lima",
+ population: 9750000,   
+}
+};
+capitalCities.UnitedKingdom = 8980000;
```

Comment:
The part related to: "Change the value of UnitedKingdom's capital city population to 8980000." You did well :)

You need to do the same way the other part of the exercise:
    - Add the property for population to China's capital city and set the value to 21500000.
    - Add the country "Peru" to capitalCities object.
    - Add a name of "Lima" to Peru's capital city.
    - Add a population of 9750000 to Peru's capital city.
    
    Please check this: https://bobbyhadz.com/blog/javascript-add-key-value-pair-to-object